examples/asset/loading_screen.rs

```diff
@@ -0,0 +1,220 @@
+use bevy::prelude::*;
+
+/// This example illustrates, how a loading screen can be implemented. It has an animated spinner and listens for an `AssetEvent`.
+
+fn main() {
+    App::new()
+        .insert_resource(Msaa { samples: 4 })
+        .add_plugins(DefaultPlugins)
+        // Loading screen plugin and its `asset_listening_system`
+        .add_plugin(plugin::SimpleLoadingScreenPlugin)
+        // system that sets up everything for our game
+        .add_startup_system(setup_game)
+        // system that will be run after everything is set up
+        .add_system_set(SystemSet::on_enter(plugin::LoadingState::Done).with_system(game_system))
+        .run();
+}
+
+// Here we prepare our gaming scene, meaning we setup our ingame camera and load our asset.
```

Comment:
```suggestion
/// Here we prepare our gaming scene, meaning we setup our ingame camera and load our asset.
```

Comments about items should be doc comments.